<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Binary Viewer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="HintLabel">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Drag and drop or &quot;Start -&gt; Open&quot; any File</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="OpenFile"/>
    <addaction name="OpenInNewWindow"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutBiosViewer"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExtract_BIOS"/>
    <addaction name="actionReplace_BIOS"/>
    <addaction name="actionSeperate_Binary"/>
    <addaction name="actionExtract_Binary"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuShow">
    <property name="title">
     <string>Show</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionGoto"/>
    <addaction name="actionCollapse"/>
    <addaction name="actionHex_View"/>
    <addaction name="actionBios_View"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuShow"/>
   <addaction name="menuTool"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenFile">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/open.svg</normaloff>:/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAboutBiosViewer">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/about.svg</normaloff>:/about.svg</iconset>
   </property>
   <property name="text">
    <string>About BIOS Viewer</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/about.svg</normaloff>:/about.svg</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/Exit.svg</normaloff>:/Exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="OpenInNewWindow">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/open.svg</normaloff>:/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open in new window</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/gear.svg</normaloff>:/gear.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionSeperate_Binary">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/scissors.svg</normaloff>:/scissors.svg</iconset>
   </property>
   <property name="text">
    <string>Separate Binary</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionExtract_BIOS">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/box-arrow-up.svg</normaloff>:/box-arrow-up.svg</iconset>
   </property>
   <property name="text">
    <string>Extract BIOS</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/search.svg</normaloff>:/search.svg</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/bookmark.svg</normaloff>:/bookmark.svg</iconset>
   </property>
   <property name="text">
    <string>Goto Offset</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionCollapse">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/arrows-collapse.svg</normaloff>:/arrows-collapse.svg</iconset>
   </property>
   <property name="text">
    <string>Collapse all</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionReplace_BIOS">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/replace.svg</normaloff>:/replace.svg</iconset>
   </property>
   <property name="text">
    <string>Replace BIOS</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionExtract_Binary">
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/box-arrow-up.svg</normaloff>:/box-arrow-up.svg</iconset>
   </property>
   <property name="text">
    <string>Extract Binary</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionHex_View">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../logo/Images.qrc">
     <normaloff>:/file-binary.svg</normaloff>:/file-binary.svg</iconset>
   </property>
   <property name="text">
    <string>Hex View</string>
   </property>
  </action>
  <action name="actionBios_View">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bios View</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../logo/Images.qrc"/>
 </resources>
 <connections/>
</ui>
